terraform/environments/electronic-monitoring-data/lambdas_main.tf

```diff
@@ -282,3 +282,32 @@ module "load_json_table" {
   }
 }
 
+
+#-----------------------------------------------------------------------------------
+# Load json data from S3 to Athena
+#-----------------------------------------------------------------------------------
+
+module "load_json_table_cadt" {
+  source                  = "./modules/lambdas"
+  function_name           = "load_json_table_cadt"
+  is_image                = true
+  role_name               = aws_iam_role.load_json_table.name
+  role_arn                = aws_iam_role.load_json_table.arn
+  memory_size             = 2048
+  timeout                 = 900
+  env_account_id          = local.env_account_id
+  core_shared_services_id = local.environment_management.account_ids["core-shared-services-production"]
+  production_dev          = local.is-production ? "prod" : "dev"
+  ecr_repo_name           = "create-a-data-task"
+  function_tag            = "v0.0.0-884806f"
```

Comment:
oooh this is new since I was last in this space

Reply:
just so i could use a docker image that wasnt in the em repo